.circleci/config.yml

```diff
@@ -15,7 +15,11 @@ jobs:
       - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
       - run: npm install
-      - run: npm run compile
+      # Setup an NODE_ENV to production mode before compiling.
```

Comment:
Update circleci builds so that the vsix files are compiled in production mode. This means we run webpack in production mode.